nservicebus/operations/performance-counters-note.include.md

```diff
@@ -0,0 +1 @@
+**Versions 6 and above:** The value recorded in the TimeSent header is the time when the call to send the message is executed, not the actual time when the message was dispatched to the transport infrastructure. Since the outgoing messages in handlers are sent as a [batched](/nservicebus/messaging/batched-dispatch.md) operation, depending on how long the message handler takes to complete, the actual dispatch may happen later than the time recorded in the TimeSent header. For operations outside of handlers the recorded sent time is accurate.
```

Comment:
> sent as a [batched](/nservicebus/messaging/batched-dispatch.md) operation

in [batches](/nservicebus/messaging/batched-dispatch.md)? or are [batched](/nservicebus/messaging/batched-dispatch.md) for sending?